packages/@sanity/color-input/src/ColorPicker.js

```diff
@@ -1,10 +1,25 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import Button from 'part:@sanity/components/buttons/default'
 import {ColorWrap, Checkboard, Saturation, Hue, Alpha} from 'react-color/lib/components/common'
-import ColorPickerFields from './ColorPickerFields'
-import TrashIcon from 'part:@sanity/base/trash-icon'
-import styles from './ColorPicker.css'
+import {Box, Card, Flex, Button, Inline, Stack, Text} from '@sanity/ui'
+import {TrashIcon} from '@sanity/icons'
+import styled from 'styled-components'
+import {ColorPickerFields} from './ColorPickerFields'
+
+const ColorBox = styled(Box)`
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+`
+
+const ReadOnlyContainer = styled(Flex)`
+  margin-top: 6rem;
+  background-color: ${({theme}) => theme.sanity.color.base.bg};
```

Comment:
I believe you can use `background-color: var(--card-bg-color)` to achieve the same thing?

Reply:
Yes, that works as well! Is it preferable to use css variables instead of accessing the theme as I did?

Reply:
I had the same question on my pr as well! ☝️